pkg/cli/newapp/newapp.go

```diff
@@ -369,7 +369,18 @@ func (o *AppOptions) RunNewApp() error {
 	if len(result.Name) > 0 {
 		// only set the computed implicit "app" label on objects if no object we've produced
 		// already has the "app" label.
-		appLabel := map[string]string{"app": result.Name}
+		appLabel := map[string]string{
+			"app":                         result.Name,
+			"app.kubernetes.io/instance":  result.Name,
+			"app.kubernetes.io/component": result.Name,
```

Comment:
From what I see you're also missing `app.kubernetes.io/part-of` label that our webconsole is setting and is mentioned in k8s docs.

Reply:
Since in `oc new-app ..`, we only take in the name of the 'component', I thought it's a good idea to not populate the `part-of` label since that label indicates the 'application group' the imported component is in.

I don't wish to give users an idea that component name == application name :)

Reply:
Well, we already kinda do it in the web console.

Reply:
True, that's what we did initially.
Then we stopped doing so of late, by supporting 'unassigned'.

![image](https://user-images.githubusercontent.com/545280/66428417-fc39b880-e9e3-11e9-899c-24c5b7db252e.png)

Reply:
Gotcha, in that case get the author squash the changes and we can land it.